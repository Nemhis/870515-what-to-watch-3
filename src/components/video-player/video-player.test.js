import React from 'react';
import renderer from 'react-test-renderer';

import VideoPlayer from './video-player.jsx';

const film = {
  previewImg: `/picture/src.jpg`,
  previewVideoLink: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
};

it(`Render correctly <VideoPlayer>`, () => {
  const options = {
    createNodeMock: () => ({}),
  };

  const tree = renderer
    .create(
        <VideoPlayer
          isPlaying={false}
          src={film.previewVideoLink}
          poster={film.previewImg}
          width={`100`}
          height={`100`}
        />, options
    ).toJSON(`test`);

  expect(tree).toMatchSnapshot();
});
