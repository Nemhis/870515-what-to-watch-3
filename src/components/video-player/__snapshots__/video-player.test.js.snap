// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly <VideoPlayer> 1`] = `
<video
  autoPlay={false}
  height="100"
  muted={false}
  poster="/picture/src.jpg"
  src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
  width="100"
/>
`;
